src/main/java/racingcar/model/RaceResults.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RaceResults {
+
+    private final List<RaceResult> raceResults;
+    private final RaceResult lastRaceResult;
+    private final List<String> winners;
+    private int maxPosition;
+
+    public RaceResults(List<RaceResult> raceResults) {
+        this.raceResults = new ArrayList<>(raceResults);
+        this.lastRaceResult = raceResults.get(raceResults.size() - 1);
+        this.winners = new ArrayList<>();
+        this.maxPosition = 0;
+    }
+
+    public List<String> findWinners() {
+        List<Car> carsAfterLastRace = lastRaceResult.getRaceResultByRound();
+        findMaxPosition(carsAfterLastRace);
+        findCarsWithMaxPosition(carsAfterLastRace);
+        return winners;
+    }
+
+    public void findMaxPosition(List<Car> carsAfterRace) {
+        for (Car car : carsAfterRace) {
+            maxPosition = car.compareMaxPosition(maxPosition);
+        }
+    }
```

Comment:
maxPosition을 인스턴스 변수로 둘 필요 없이, 지금처럼 지역변수로 대체할 수 있을 것 같아요.
또한 메서드의 반환 값으로 얻을 수 있다면, 필요한 시점에 한 번만 계산해서 maxPostion 결과를 얻어올 수 있을 것 같아요. 🤗

```suggestion
    public int findMaxPosition(List<Car> carsAfterRace) {
        int maxPosition = 0;
        for (Car car : carsAfterRace) {
            maxPosition = car.compareMaxPosition(maxPosition);
        }

        return maxPosition;
    }
```